azkaban_cli/azkaban_cli.py

```diff
@@ -438,6 +440,16 @@ def __fetch_executions_of_a_flow(ctz, project, flow, start, length):
     except FetchExecutionsOfAFlowError as e:
         logging.error(str(e))
 
+@login_required
+def __pause_flow_execution(ctx, exec_id):
+    azkaban = ctx.obj[u'azkaban']
+
+    try:
+        json = azkaban.pause_flow_execution(exec_id)
+        logging.info(json)
```

Comment:
You don't need to log anything because the response is empty (`{}`) in case of success. Maybe we could log a success message, but then we would need to do so for all other responses that don't necessarily return anything.